adb_graphics/default_specs.yml

```diff
@@ -1355,29 +1355,29 @@ ulwrf: # Upward Longwave Radiation Flux
     colors: radiation_colors
     ncl_name: ULWRF_P0_L1_{grid}
     ticks: -2
-    title: Upward Longwave Radiation Flux, Surface
+    title: Upward LW Radiation Flux, Surface
```

Comment:
I think it would be more clear to abbreviate `surface` to `sfc` than `Longwave` here. Same on the variables below.